tests/syslog/test_syslog.py

```diff
@@ -0,0 +1,58 @@
+import time
+import logging
+import pytest
+from common.helpers.assertions import pytest_assert
+
```

Comment:
Please add topology marker to this test case, it should be topology any.

Reply:
Added in the latest commit